tpl/tplimpl/embedded/templates/_default/rss.xml

```diff
@@ -10,6 +10,7 @@
 {{- if ge $limit 1 -}}
 {{- $pages = $pages | first $limit -}}
 {{- end -}}
+{{- $lastModPage := (sort $pages "Lastmod" "desc") | first 1 }}
```

Comment:
There is a `.ByLastmod` method on the `Pages` type.

Reply:
But, I would prefer if we could calculate this date by looping instead of sorting.

Reply:
I updated the condition to `with index $pages.ByLastmod.Reverse 0` and removed this `$lastModPage` definition which seems the cleanest solution.

```
{{- with index $pages.ByLastmod.Reverse 0 -}}{{- if not .Lastmod.IsZero -}}
    <lastBuildDate>{{ .Lastmod.Format "Mon, 02 Jan 2006 15:04:05 -0700" | safeHTML }}</lastBuildDate>
{{- end -}}{{- end -}}
```